portfolio/src/main/webapp/index.html

```diff
@@ -2,56 +2,275 @@
+        <div class="carousel-indicators">
+          <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+          <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
+          <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
+        </div>
+        <div class="carousel-inner">
+          <div class="carousel-item active">
+            <img src="./images/SF.png" class="d-block w-100" alt="June 2022 in San Fransisco">
+            <div class="carousel-caption d-none d-md-block">
+              <h5>Golden Bridge</h5>
+              <p>June 2022 in San Fransisco</p>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./images/fluorescent_sea.JPG" class="d-block w-100" alt="fluorescent_sea">
+            <div class="carousel-caption d-none d-md-block">
+              <h5>Fluorescent Sea</h5>
+              <p>March 2022 in San Deigo</p>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./images/BigSur_Sunset.png" class="d-block w-100" alt="BigSur_Sunset">
+            <div class="carousel-caption d-none d-md-block">
+              <h5>Sunset</h5>
+              <p>June 2022 in Big Sur</p>
+            </div>
+          </div>
+        </div>
+        <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="visually-hidden">Previous</span>
+        </button>
+        <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="visually-hidden">Next</span>
+        </button>
+      </div>
+      <p></p>
+      <div class="center">
+        <p style="font-family:Rockwell; font-size: 30px;">MORE INTERACTION WITH ME!</p>
+      </div>
     </div>
+        <div id="interaction">
+            <p style="font-family:Comic Sans MS">Click this button to see the server's current date.</p>
+            <button type="button" class="btn btn-outline-primary" onclick="showServerTime();" style="font-family:Comic Sans MS">
+            Get date
+            </button>
+            <p id="date-container"></p>
+        </div>
+        <div id="interaction">
+            <form action="/form-handler" method="POST">
+            <p style="font-family:Comic Sans MS">Text me anything:</p>
+            <textarea name="text-input">Hi!</textarea>
+            <br/>
+            <input type="submit" class="btn btn-outline-dark"/>
+            </form>
+        </div>
+    </div>
+  </body>
+  <body onload="loadContacts()">
```

Comment:
Curious, why use multiple `<body>` tags within a single page?

Reply:
aha, forget to change this. I think I just follow the walkthrough demo which just use <body> tags in the example. I will merge all <body> into one single <body>.